sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1029,8 +1029,11 @@ def _retry_region_zones(self,
                                                       cluster_exists):
             if self._in_blocklist(to_provision.cloud, region, zones):
                 continue
-            zone_str = ','.join(
-                z.name for z in zones) if zones is not None else 'all zones'
+            if zones is None or len(zones) == 0:
```

Comment:
style nit: How about? reference: https://google.github.io/styleguide/pyguide.html#2144-decision
```suggestion
            if not zones:
```

Reply:
Thanks for letting me know. Fixed.